install/envs/mac.yml

```diff
@@ -25,6 +26,14 @@ dependencies:
   - PrettyTable
   - pyfiglet
   - mypy
+  - pytorch=1.7.1
+  - torchvision
+  - torchaudio
+  - docopt
+  - pytorch-lightning
+  - imgaug
+  - matplotlib
+  - numpy
```

Comment:
I think we forgot this as it always comes as a dependency of pandas or tf, but better to put it here!

Reply:
Just pushed with fixes